DigitalLearningSolutions.Web/Views/TrackingSystem/Delegates/AllDelegates/_SearchableDelegateCard.cshtml

```diff
@@ -7,7 +7,8 @@
   <details class="nhsuk-details nhsuk-expander">
     <summary class="nhsuk-details__summary">
       <span class="nhsuk-details__summary-text" id="@Model.DelegateInfo.Id-name">
-        <span class="searchable-element-title" name="name">@Model.DelegateInfo.TitleName</span> <span class="searchable-element-candidate-number">(@Model.DelegateInfo.CandidateNumber)</span>
+        <span class="searchable-element-title" name="name">@Model.DelegateInfo.TitleName</span>
+        <span class="searchable-element-candidate-number nhsuk-u-visually-hidden">@Model.DelegateInfo.CandidateNumber</span>

```

Comment:
I agree with Dan that the change as is causes the search functionality between JS and no JS to diverge. I also think that this visually hidden span can probably be removed in favour of adding the class to the Candidate number display within the card.

This should probably be checked with HEE whether they want searching on the email or not.